rfd/0155/README.md

```diff
@@ -10,741 +10,11 @@ discussion: 'https://github.com/joyent/rfd/issues?q=%22RFD+155%22'
 -->
 
 <!--
-    Copyright (c) 2019, Joyent, Inc
+    Copyright 2019 Joyent, Inc
 -->
 
 # RFD 155 Manta Buckets API
 
-This RFD will describe the new buckets API for the Manta storage service. The
-functional purpose of this requires some background understanding of Manta's
-capabilities today.
+Documentation for the Manta Buckets API has been made official:
```

Comment:
Do you think we should add a note that says something like 'the buckets API was originally tracked in this RFD (see revision history) as we were going through design review. The initial design is now official. Further changes are made here: [link]'?

Reply:
I like that... i'll comment that exactly